packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -151,6 +151,30 @@ export class Loan {
     return await this.client.call('setloantoken', [{ ...defaultData, ...loanToken }, utxos], 'number')
   }
 
+  /**
+   * Updates an existing loan token.
+   *
+   * @param {UpdateLoanToken} loanToken
+   * @param {string} loanToken.token Previous tokens's symbol, id or creation tx (unique)
+   * @param {string} loanToken.symbol New token's symbol (unique), no longer than 8
+   * @param {string} [loanToken.name] Token's name, no longer than 128
+   * @param {string} loanToken.priceFeedId Txid of oracle feeding the price
+   * @param {boolean} [loanToken.mintable = true] Token's 'Mintable' property
+   * @param {BigNumber} [loanToken.interest = 0] Interest rate
```

Comment:
all fields are optional.

Reply:
Why do you say so?

Reply:
it's update loan token. so you can basically give any combination of fields that needs to be updated.
Ref- https://github.com/DeFiCh/ain/blob/4492dc3d7a3debd368774e3e2707eb9e208a6a6b/src/masternodes/rpc_loan.cpp#L375
Also, you can check how the CPP side RPC is written there.